lib/urlwatch/filters.py

```diff
@@ -183,6 +183,20 @@ def filter(self, data, subfilter=None):
         return ical2text(data)
 
 
+class JsonFormatFilter(FilterBase):
+    """Convert to formatted json"""
+
+    __kind__ = 'formatted-json'
+
+    def filter(self, data, subfilter=None):
+        from json import dumps
```

Comment:
Since the `json` module ships with Python 3, please do the `import json` at the top of the file, and use `json.loads()` / `json.dumps()` here.

Reply:
Done. I didn't really understand why some filters had imports in the body and some imports were at the top of the file. Thanks for giving a plausible explanation for that practise :)